parquet-benchmarks/README.md

```diff
@@ -25,10 +25,29 @@ First, build the ``parquet-benchmarks`` module
 mvn --projects parquet-benchmarks -amd -DskipTests -Denforcer.skip=true -P hadoop-2 clean package
```

Comment:
I don't know what has changed in the mean time, but with the `hadoop-2` profile now the generated parquet-benchmarks jar doesn't contain `/META-INF/BenchmarkList` and it fails

```
[/Users/nyigitbasi/other_workspace/parquet-mr/parquet-benchmarks] >> ./run.sh -wi 1 -i 1 -f 1 -p parquetVersion=v2 -p randomData=true
Starting WRITE benchmarks
Exception in thread "main" java.lang.RuntimeException: ERROR: Unable to find the resource: /META-INF/BenchmarkList
	at org.openjdk.jmh.runner.AbstractResourceReader.getReaders(AbstractResourceReader.java:96)
	at org.openjdk.jmh.runner.BenchmarkList.find(BenchmarkList.java:104)
	at org.openjdk.jmh.runner.Runner.internalRun(Runner.java:228)
	at org.openjdk.jmh.runner.Runner.run(Runner.java:178)
	at org.openjdk.jmh.Main.main(Main.java:66)
```

A build without specifying any profile creates a valid jar though. So build command should be

```
mvn --projects parquet-benchmarks -amd -DskipTests -Denforcer.skip=true clean package
```